vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -1281,15 +1292,36 @@ public Component getListCellRendererComponent(
       // It is incredibly inefficient for GamePieces and is not needed
       // since we overwrite the label text anyway.
       super.getListCellRendererComponent(list, "", index, selected, hasFocus);
+
+      final int lineOffset = definer.isPrototype() ? 1 : 0;
+
       if (value instanceof EditablePiece) {
-        setText(((EditablePiece) value).getDescription());
+        final EditablePiece editableValue = (EditablePiece) value;
+        final String type = editableValue.getType().split(";")[0];
+
+        // Prototype? Skip first (dummy) item to avoid outputting a blank line
+        if (lineOffset == 1 || index > 0) {
+          // For Pieces, bump index to 1
+          // Special formatting for Comments
+          if (type.startsWith("cmt")) {
```

Comment:
Better would be `if (type.startsWith(Commend.ID))`

Reply:
Done, accounting for the trailing separator in Comment.ID.